.gitignore

```diff
@@ -1,42 +1,51 @@
 # Byte-compiled / optimized / DLL files
 __pycache__/
-*.py[cod]
+/*.py[cod]
+/src/PIL/*.py[cod]
+/Tests/*.py[cod]
+/docs/*.py[cod]
+/winbuild/*.py[cod]
 
 # C extensions
-*.so
+/src/PIL/*.so
```

Comment:
If `.so` is included, perhaps also include `.dll`? Although they are hidden by my global `.gitignore`, this might not be the case for others.